portfolio/src/main/java/com/google/sps/data/Reply.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+/** A comment on the forum page. */
+public final class Reply {
+
+  private final long id;
+  private final String name;
+  private final String comment;
+  private final String email;
+  private final String date;
+  private final long parentId;
+
+  public Reply(long id, String name, String comment, String email, String date, long parentId) {
```

Comment:
This looks a lot like the Comment class but has an additional field parentId. Do you think it makes sense to have Reply take (1) Comment and (2) parentId?

Reply:
fixed it by just passing 0 as parentId for comments

Reply:
I don't think I see this code updated yet. Could you check that out? Do you mean you initialized Comment to be a Reply with parentId 0? 

Also, see the other comments in this PR as well.